Formula/tektoncd-cli.rb

```diff
@@ -0,0 +1,24 @@
+class TektoncdCli < Formula
+  desc "CLI for interacting with TektonCD"
+  homepage "https://github.com/tektoncd/cli"
+  url "https://github.com/tektoncd/cli/archive/v0.2.2.tar.gz"
+
+  sha256 "5a8c149a23cfe4c536b8fbefefd0d04618b821d4adc518d564ebd3f0068df5be"
+
+  depends_on "go" => :build
+
+  def install
+    system "make", "bin/tkn"
```

Comment:
maybe explicit golang source build would be easier?

This seems little bit hacky?

```
bin/%: cmd/% ./vendor FORCE
	@go build -v -o $@ ./$<
```

Reply:
Having a makefile makes it way easier to get builds the same as upstream would expect. If there is a makefile you should use it.